packages/server/utils/OpenAIServerManager.ts

```diff
@@ -21,7 +21,11 @@ class OpenAIServerManager {
     try {
       const response = await this.openAIApi.createCompletion({
         model: 'text-davinci-003',
-        prompt: `Summarize this for a second-grade student in one or two sentences: ${text}`,
+        prompt: `Below is a comma-separated list of text. Summarize the text for a second-grade student in one or two sentences.
```

Comment:
+1 We could give it even more hints for summarising reflections:

> Below is a list of topics with bullet points. Summarize these for a second-grade student in one or two sentences:
> 
> What went well?
> - We improved the prompt quality
> 
> What was difficult?
> - getting everyone on the same page

Reply:
FYI my original hack day [implementation](https://github.com/ParabolInc/parabol/blob/feat/gpt3Summaries/packages/client/utils/OpenAIManager.ts) on the prompt is similar to what Georg suggested.

Reply:
Sounds good - merging this PR for now so Georg can continue the ship and will play with the suggested prompt later